### ISSUE TITLE
SASRec performance is inferior than BPR-MF!

### ISSUE THREAD

**Issue by ltz0120** [kang205/SASRec#13]
I find the performance of SASRec is inferior than BPR-MF on some sparse dataset like Amazon beauty, game, and steam dataset. Is it normal? I use the BPR-MF baseline form https://github.com/duxy-me/ConvNCF.

**Comment by kang205** [comment]
It's not normal, SASRec should significantly outperform BPR-MF on the datasets you mentioned. This is also observerd in many many other follow-up works like BERT4Rec (https://arxiv.org/pdf/1904.06690.pdf).

**Comment by BEbillionaireUSD** [comment]
Hi, I calculated the "real hit rate", which is to predict the probability of all items appearing in the next-item position. The training hit rate@10 of the movielen-1m dataset is only 0.15. Is it normal? (epochs 800)

If I split it into a training set and a test set, the top 10 hit rate is only 0.03. It seems to be extremely low.

**Comment by conquerSelf** [comment]
> It's not normal, SASRec should significantly outperform BPR-MF on the datasets you mentioned. This is also observerd in many many other follow-up works like BERT4Rec (https://arxiv.org/pdf/1904.06690.pdf).

I have the same question.Can you provide me with the baseline model code?

**Comment by youngzw** [comment]
> Hi, I calculated the "real hit rate", which is to predict the probability of all items appearing in the next-item position. The training hit rate@10 of the movielen-1m dataset is only 0.15. Is it normal? (epochs 800)
> 
> If I split it into a training set and a test set, the top 10 hit rate is only 0.03. It seems to be extremely low.

I got the similar result with ranking all items.
